apps/src/templates/sectionProgress/progressTables/ProgressTableLevelIcon.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {levelTypeWithoutStatus} from '@cdo/apps/templates/progress/progressTypes';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import color from '@cdo/apps/util/color';
+import * as progressStyles from '@cdo/apps/templates/progress/progressStyles';
+import {getIconForLevel} from '@cdo/apps/templates/progress/progressHelpers';
+
+const styles = {
+  container: {
+    ...progressStyles.flex,
+    ...progressStyles.cellContent
+  },
+  icon: {
+    ...progressStyles.inlineBlock,
+    textAlign: 'center',
+    color: color.charcoal,
+    fontSize: 20
+  }
+};
+
+function levelIcon(level) {
+  return (
+    <span
+      style={{
+        ...styles.icon,
+        width: progressStyles.BUBBLE_CONTAINER_WIDTH
+      }}
+    >
+      <FontAwesome icon={getIconForLevel(level, true)} />
+    </span>
+  );
+}
+
+// extra space is rendered in the columns with sublevels
+// so that the header will align properly with the student progress
+function sublevelSpace(sublevelCount) {
```

Comment:
(minor) Not sure what others think, but I like to name functions that return a React element `renderFoo()` so this function might be called `renderSublevelSpace()`.  Up to you.

Reply:
i support adopting this as a convention!